plugins/Live/templates/_totalVisitors.twig

```diff
@@ -1,29 +1,9 @@
-<div id="visitsTotal">
-    <table class="dataTable" cellspacing="0">
-        <thead>
-        <tr>
-            <th id="label" class="sortable label first" style="cursor: auto;">
-                <div id="thDIV">{{ 'General_Date'|translate }}</div>
-            </th>
-            <th class="sortable" style="cursor: auto;">
-                <div id="thDIV">{{ 'General_ColumnNbVisits'|translate }}</div>
-            </th>
-            <th class="sortable" style="cursor: auto;">
-                <div id="thDIV">{{ 'General_Actions'|translate }}</div>
-            </th>
-        </tr>
-        </thead>
-        <tbody>
-        <tr class="">
-            <td class="label column">{{ 'Live_LastHours'|translate(24) }}</td>
-            <td class="column" {% if countErrorToday %}title="{{ countErrorToday|e('html_attr') }}"{% endif %}>{{ visitorsCountToday|number }}</td>
-            <td class="column" {% if countErrorToday %}title="{{ countErrorToday|e('html_attr') }}"{% endif %}>{{ pisToday|number }}</td>
-        </tr>
-        <tr class="">
-            <td class="label column">{{ 'Live_LastMinutes'|translate(30) }}</td>
-            <td class="column" {% if countErrorHalfHour %}title="{{ countErrorHalfHour|e('html_attr') }}"{% endif %}>{{ visitorsCountHalfHour|number }}</td>
-            <td class="column" {% if countErrorHalfHour %}title="{{ countErrorHalfHour|e('html_attr') }}"{% endif %}>{{ pisHalfhour|number }}</td>
-        </tr>
-        </tbody>
-    </table>
-</div>
+<div
+    vue-entry="Live.TotalVisitors"
+    count-error-today="{{ countErrorToday|default(null)|json_encode }}"
+    visitors-count-today="{{ (visitorsCountToday + 0)|default(null)|json_encode }}"
+    pis-today="{{ pisToday|default(null)|json_encode }}"
+    count-error-half-hour="{{ countErrorHalfHour|default(null)|json_encode }}"
+    visitors-count-half-hour="{{ (visitorsCountHalfHour + 0)|default(null)|json_encode }}"
+    pis-halfhour="{{ pisHalfhour|default(null)|json_encode }}"
+></div>
```

Comment:
Just out of curiosity: Was there any good reason for converting this one into a vue component? It actually doesn't contain any logic that might require it. 
Reverting this one back into plain html fixes the updating issue for me. Guess the current problem might be that the `id="visitsTotal"` is not in the surrounding element anymore, but the live widget refresh tries to replace the element by that id. Guess that might end up in nested `Live.TotalVisitors` elements, which vue seems to prevent maybe. At least the update only works once for me and all updates afterwards don't replace the content anymore.

Reply:
@sgiehl Yes, _totalVisitors.twig is used in index.twig, which was converted to Vue, and the Vue component can't include the twig template. However, because the live widget replaces it in html it's definitely a mistake to directly use the component in Vue, as Vue doesn't expect the HTML it manages to be changed from the outside. I pushed a change that uses VueContainer in the live page. At this point, we can keep TotalVisitors as a Vue component or change it back, should be fine either way. If the goal is to eventually get rid of the jquery code, though, it might be better to just keep it in Vue.

Can you check if the change works for you?

Reply:
Yes. Now the refresh works for me 👍